.gitattributes

```diff
@@ -0,0 +1,11 @@
+/.github/ export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/Makefile export-ignore
+/assets/ export-ignore
+/composer.lock export-ignore
+/ecs.php export-ignore
+/grumphp.yml export-ignore
+/install/ export-ignore
```

Comment:
I am not sure about `install` directory.

Reply:
Indeed, `install` directory is needed here so it shouldn't be excluded:
https://github.com/phpsword/installer/blob/1521292eb0f3e05917396d6b82566a11655e63da/.github/workflows/build.yml#L78-L80

Reply:
Done! 🍏